requirements/dev.txt

```diff
@@ -1,3 +1,3 @@
-hupper>=0.4.0
+hupper==1.7
```

Comment:
```suggestion
hupper>=0.4.0,!=1.8,!=1.8.1
```
Otherwise we won't ever get a newer version.

Reply:
Sure, just thought it safe to pin it until https://github.com/Pylons/hupper/issues/54 was resolved. In case a new version is pushed without a fix. Are we ok with that scenario?

Reply:
Ah, didn't realize you had created an issue. Sure, if you think you'll remember to follow up when it's fixed, fine with me. Could you just add a comment linking to that issue to explain the pin?